src/common.rs

```diff
@@ -47,11 +47,13 @@ pub mod node_address {
     const ARG_VALUE_NAME: &str = "HOST:PORT";
     const ARG_DEFAULT: &str = "http://localhost:7777";
     const ARG_HELP: &str = "Hostname or IP and port of node on which HTTP service is running";
+    const ARG_ENV: &str = "CASPER_NODE_ADDRESS";
```

Comment:
Hi,

Can we try to avoid CASPER_ prefixes ? Is it still a node or an rpc RPC_ADDRESS ?

It would be great to unify that with CI/CD like

NODE_ADDRESS=http://localhost:7777 (or RPC_ADDRESS)
EVENT_ADDRESS=http://localhost:9999/events/main
SPECULATIVE_ADDRESS=http://localhost:7778

Reply:
to me the client should now use RPC_ADDRESS and the **binary client** NODE_ADDRESS